pudb/debugger.py

```diff
@@ -1743,23 +1728,15 @@ def cmdline_exec(w, size, key):
             prev_sys_stdout = sys.stdout
             prev_sys_stderr = sys.stderr
 
-            if PY3:
-                from io import StringIO
-            else:
-                from cStringIO import StringIO
+            from io import StringIO
 
             sys.stdin = None
             sys.stderr = sys.stdout = StringIO()
             try:
                 # Don't use cmdline_get_namespace() here in Python 2, as it
                 # breaks things (issue #166).
```

Comment:
```suggestion
```
This comment is outdated.

Reply:
Thanks for taking a look! I think 71ffa51 should address all the points you raised. As for `pudb3`, I've left that in, but I've not added a `pudb` script back. I think that, ultimately, `python -m pudb` is the more robust thing to use. Ive' also changed the docs to push people towards that.

> I can't figure out how to add a comment to lines that weren't affected by the PR in github :(

I don't think Github allows it.